src/main/java/worker/Main.java

```diff
@@ -64,12 +73,14 @@ public static void startWorker(int port) {
 
     ActorSystem system = ActorSystem.create("WorkerSystem", conf);
 
-    Set<ActorSelection> initialContacts = new HashSet<ActorSelection>();
+    Set<ActorPath> initialContacts = new HashSet<>();
     for (String contactAddress : conf.getStringList("contact-points")) {
-      initialContacts.add(system.actorSelection(contactAddress + "/user/receptionist"));
+      initialContacts.add(ActorPaths.fromString(contactAddress + "/system/receptionist"));
     }
```

Comment:
Done, it's actually just one client (the "Worker") so it was safe to put it under the default config path in `worker.con`